backend/app/crud/project.py

```
@@ -0,0 +1,164 @@
+from typing import List, Optional
+from sqlalchemy.orm import Session, joinedload
+from ..crud.base import CRUDBase
+from ..models import Project, ProjectMember, User, Task
```

Ruff (0.11.9)

`..models.User` imported but unused

Remove unused import

(F401)